src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -83,20 +83,20 @@ public function __construct(
     }
 
     /**
-     * @Given the store allows paying :paymentMethodName
-     * @Given the store allows paying with :paymentMethodName
+     * @Given the store (also) allows paying (with) :paymentMethodName
```

Comment:
That's not true, Turnip will allow for the following patterns:
```
store allows
store also allows
```

Unfortunately, it will also allow for:
```
store  allows
storeallows
storealso allows
store alsoallows
storealsoallows
```

See https://github.com/Behat/Behat/blob/v3.2.2/src/Behat/Behat/Definition/Pattern/Policy/TurnipPatternPolicy.php#L161